samples/subsys/fs/format/README.rst

```diff
@@ -0,0 +1,46 @@
+.. _fs_format_sample:
+
+FS Format Sample
+################
+
+Overview
+***********
+
+This sample shows how to format different storage
+devices for different file systems. There are 2 scenarios prepared for this
+sample:
+* littleFS on flash device
+* FAT file system on RAM disk
+
+Building and running
+********************
+
+To run this sample, build it for the desired board and scenario and flash it.
+
+The Flash scenario is supported on the nrf52dk_nrf52832 board.
+The RAM disk scenario is supported on the mimxrt1064_evk board.
+To build the RAM disk sample, the configuration `prj_ram.conf` needs to be used by setting `CONF_FILE=prj_ram.conf`.
+
+The Flash sample for the nrf 52DK board can be build as follow:
+.. zephyr-app-commands::
+	:zephyr-app: samples/subsys/fs/format
+	:board: nrf52dk_nrf52832
+	:goals: build flash
+	:compact:
+
+The RAM disk sample for the MIMXRT1064-EVK board can be build as follow:
+.. zephyr-app-commands::
+	:zephyr-app: samples/subsys/fs/format
+	:board: mimxrt1064_evk
+   :conf: "prj_ram.conf"
+	:goals: build flash
+	:compact:
```

Comment:
Indention seems broken, this is what diff in console shows:
```
+The Flash sample for the nrf 52DK board can be build as follow:
+.. zephyr-app-commands::
+       :zephyr-app: samples/subsys/fs/format
+       :board: nrf52dk_nrf52832
+       :goals: build flash
+       :compact:
+
+The RAM disk sample for the MIMXRT1064-EVK board can be build as follow:
+.. zephyr-app-commands::
+       :zephyr-app: samples/subsys/fs/format
+       :board: mimxrt1064_evk
+   :conf: "prj_ram.conf"
+       :goals: build flash
+       :compact:
```

Reply:
fixed and did some blank lines to also make it more readable.